src/dataframe/dataframe.jl

```diff
@@ -257,6 +257,32 @@ function DataFrame(column_eltypes::AbstractVector{T}, nrows::Integer) where T<:T
     return DataFrame(column_eltypes, gennames(length(column_eltypes)), nrows)
 end
 
+"""
+    DataFrame!(args...; kwargs...)
+
+Create a `DataFrame` from a `args` and `kwargs` with `copycols=false`.
```

Comment:
Maybe just "Equivalent to `DataFrame(args...; copycols=false, kwargs...)`"?

Reply:
OK. fixed